README.md

```diff
@@ -217,6 +217,15 @@ Here are the list of all the executable scripts aswell as their arguments :
 
 <br />
 
+* retrieveDB : The script allows to get power data from Cassandra database into csv files. 1 file = 1 home and 1 specific day.
```

Comment:
When opening a PR, it would be good that you merge all the commits on a file creation into one. You could use `git rebase -i --autosquash <commit before your changes>`.